videoencoder.cpp

```diff
@@ -72,11 +72,13 @@ static int theora_write_frame(th_ycbcr_buffer ycbcr, int last)
     }
 
     ogg_stream_packetin(&ogg_os, &op);
+    if (!ogg_fp) {
```

Comment:
updated, not called anymore and the "if" replaced by "assert" at start of function